environs/manual/sshprovisioner/init_test.go

```diff
@@ -120,7 +123,7 @@ func (s *initialisationSuite) TestDetectHardwareCharacteristics(c *gc.C) {
 		c.Logf("test %d: %s", i, test.summary)
 		scriptResponse := strings.Join(test.scriptResponse, "\n")
 		defer installFakeSSH(c, sshprovisioner.DetectionScript, scriptResponse, 0)()
-		hc, _, err := sshprovisioner.DetectSeriesAndHardwareCharacteristics("hostname")
+		hc, _, err := sshprovisioner.DetectBaseAndHardwareCharacteristics("hostname")
```

Comment:
I'd also check the returned base (here and in every test using `DetectBaseAndHardwareCharacteristics`), because you are passing the base as the output of the script.

Reply:
Discussed over a meet

There is another test for this, so we're happy